tests/NLog.UnitTests/Layouts/JsonLayoutTests.cs

```diff
@@ -441,6 +442,28 @@ public void IncludeAllJsonProperties()
             Assert.Equal(ExpectedIncludeAllPropertiesWithExcludes, jsonLayout.Render(logEventInfo));
         }
 
+        [Fact]
+        public void ExcludeEmptyJsonProperties()
+        {
+            var jsonLayout = new JsonLayout()
+            {
+                IncludeAllProperties = true,
```

Comment:
Please change from `IncludeAllProperties` (deprecated in NLog 5.0) to `IncludeEventProperties`